### ISSUE TITLE
remember which channels were showing bots

### ISSUE THREAD

**Issue by silentwings** [springlobby/springlobby#511]
(useful for autohost admins - often there is a dedicated channel for each autohost where all those bots sit)

**Comment by abma** [comment]
use the channel list? give it a useful topic?

**Comment by silentwings** [comment]
huh? i don't understand.

**Comment by nixtux** [comment]
I agree the way it is now is not what a host admin want (I even joined with new lobby and restarted my hosts thinking they were offline), can we at least have it remember selection from user if he wants bots hidden or not between lobby restart

**Comment by abma** [comment]
oh, rephrased description, so its understandable by abma (tm)
